generate_stats.py

```diff
@@ -73,20 +87,28 @@ def collect_statistics(my_core, root_set):
     entityset_ranges = dagmc_stats.get_entityset_ranges(my_core, root_set, dagmc_tags['geom_dim'])
     
     spv_key = 'S_P_V'
+    tps_key = 'T_P_S'
     
-    data[spv_key] = dagmc_stats.get_surfaces_per_volume(my_core, entityset_ranges)
-    stats[spv_key] = get_stats(data[spv_key])
-    
+    data[spv_key] = dagmc_stats.get_surfaces_per_volume(
+                                my_core, entityset_ranges)
+    data[tps_key] = dagmc_stats.get_triangles_per_surface(
```

Comment:
I think it may be best to keep all the code for each statistic grouped together.  So each statistic will be roughly:
(peudocode)
```
some_key = 'key_string'
data[some_key] = dagmc_stats.some_function(....)
stats[some_key] = get_stats(data[some_key].values())
```